### ISSUE TITLE
Can not pass dart-define custom args

### ISSUE THREAD

**Issue by joecks** [dotdoom/fastlane-plugin-flutter#29]
The args section breaks dart-define custom args. 
Example:

```ruby
flutter(args: ["build", "apk", "--dart-define=SOME_VAR=\"myvalue\""])
```

causes escaping of the equals sign which breaks the arguments.

**Comment by joecks** [comment]
> GNU bash, version 5.1.8(1)-release (aarch64-apple-darwin20.4.0)

But thanks for the fast reply it made me thinking, and I tried it again, and finally I fixed it but I found it quite difficult to understand why the build was failing. 

So my issues were:
- when transitioning from flutter bash command to the flutter action, I basically took my command and uses %W(...) to create the array.
- Then I had an empty variable that I have passed to the array, which is then added as an empty string litteral:
```ruby
  "flutter build #{codesign} ..." => flutter(args: %W(build #{codesign} ...)) => flutter build '' ... 
```
which would fail in case of an empty variable
- Then the --dart-defines would pass quotes as part of the value to the bash 

```bash
 --dart-define\=USER_PROVISIONING_PROFILE\=\"Internal\ Adhoc\ Profile\"
 ```
 
 because I appended them like that:
 
 ```ruby
 flutter( args: ["--dart-define=USER_PROVISIONING_PROFILE=\"Internal\ Adhoc\ Profile\""] )
 ```
 
 So iOS could find the profile with the name '"Internal Adhoc Profile"'
 
 Long story short it works now, but I found it a bit difficult to figure it out. Just as a suggestion maybe add a dart-define param to the flutter_build command that would accept a Map of key + value pairs.
